src/styles/css/index.css

```diff
@@ -38,15 +38,14 @@ body, html {
 .App {
```

Comment:
Generated file should go probably in a `dist` dir and being ignored. What do you think?

Reply:
It already is ignored. I just removed the two files as they are not needed anymore.